src/elf.rs

```diff
@@ -107,6 +107,7 @@ impl From<GoblinError> for ElfError {
             GoblinError::BadMagic(magic) => Self::FailedToParse(format!("bad magic: {:#x}", magic)),
             GoblinError::Scroll(error) => Self::FailedToParse(format!("read-write: {}", error)),
             GoblinError::IO(error) => Self::FailedToParse(format!("io: {}", error)),
+            _ => Self::FailedToParse("cause unkown".to_string()),
```

Comment:
What is the uncovered error here?

Reply:
Goblin added a new error type and a non-exhaustive label, added a catch-all here.  I think we probably need to featureize this and add in a new error type.  This is a place holder while you looked at the other dep changes to see if you have any other concerns